application/core/calendar/calendarInvite.go

```diff
@@ -110,15 +112,41 @@ func SendInvite(matches []matcher.Match) (string, error) {
 				}
 			case mailErrors := <-errorschannel:
 				fmt.Println("received", mailErrors)
+
 				if len(mailErrors) > 0 {
 					database.UpdateJobErrors(jobId, mailErrors)
 				}
+				_, err := saveMatchingInfo(len(matches), len(mailErrors))
+
+				if err != nil {
+					mailErrors = append(mailErrors, err.Error())
+					database.UpdateJobErrors(jobId, mailErrors)
+				}
 			}
 		}
 	}()
 	return jobId, nil
 }
 
+func saveMatchingInfo(numGroups int, numFailed int) (string, error) {
+	numConversations := numGroups - numFailed
+
+	matchingStat := entity.MatchingStat{
+		NumGroups:        numGroups,
+		NumConversations: numConversations,
+		NumFailed:        numFailed,
+	}
+
+	MatchingId, err := database.CreateMatchingStat(matchingStat)
+
+	if err != nil {
+		return "", err
+	} else {
```

Comment:
you do not need if else at line 140 do directly return database.CreateMatchingStat(matchingStat)